plugin/builtin/prefixtransformer/PrefixTransformer.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:generate pluginator
+package main
+
+import (
+	"errors"
+	"sigs.k8s.io/kustomize/api/filters/prefix"
+
```

Comment:
move `"sigs.k8s.io/kustomize/api/filters/prefix"` down with the other `sigs` imports.

please audit your other files for the import style, I may have missed some